fabric-common-tags-api-v1/src/main/java/net/fabricmc/fabric/api/tag/v1/CommonBiomeTags.java

```diff
@@ -0,0 +1,83 @@
+/*
+ * Copyright (c) 2016, 2017, 2018, 2019 FabricMC
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package net.fabricmc.fabric.api.tag.v1;
+
+import net.minecraft.tag.TagKey;
+import net.minecraft.world.biome.Biome;
+
+import net.fabricmc.fabric.impl.tag.common.TagRegistration;
+
+/**
+ * See {@link net.minecraft.tag.BiomeTags} for vanilla tags.
+ * Note that addition to some vanilla tags implies having certain functionality.
+ */
+public class CommonBiomeTags {
+	public static final TagKey<Biome> IN_OVERWORLD = register("in_overworld");
+	// The following are based on Biome categories, see Biome$Category for details
+	public static final TagKey<Biome> IN_THE_END = register("in_the_end");
+	public static final TagKey<Biome> IN_NETHER = register("in_nether");
+	public static final TagKey<Biome> TAIGA = register("taiga");
+	public static final TagKey<Biome> EXTREME_HILLS = register("extreme_hills");
+	public static final TagKey<Biome> JUNGLE = register("jungle");
+	public static final TagKey<Biome> MESA = register("mesa");
+	public static final TagKey<Biome> PLAINS = register("plains");
+	public static final TagKey<Biome> SAVANNA = register("savanna");
+	public static final TagKey<Biome> ICY = register("icy");
+	public static final TagKey<Biome> BEACH = register("beach");
+	public static final TagKey<Biome> FOREST = register("forest");
+	public static final TagKey<Biome> OCEAN = register("ocean");
```

Comment:
Wouldn't it be good to include a deep ocean tag and *maybe* a shallow ocean tag to differentiate both?